### PR TITLE
Add content for kustomize

### PR DESCRIPTION
**Pull request by r0ckarong**
Still missing more general details what it does and what users should pay attention to. 

Added a note about how to generate the addon configs for an existing cluster after this discussion: 
https://chat.suse.de/channel/caasp-release-coordination?msg=ZqDmtPGAM2dWq7k8T

@ereslibre Can you amend more info on kustomize?
@jordimassaguerpla Please verify that the info I've provided is correct.

Thanks

cc / @kravciak @mjura

### CODE REVIEW

File: adoc/MAIN.release-notes.adoc [hunk 1]
```diff
@@ -23,6 +23,34 @@ To make such entries easier to identify, they contain a note to that effect.
...
+
+You must convert your addon manifests to the new `kustomize` aware file structure and formats.
+To do so please run the following command from your management workstation.
+
+Replace `MASTER-NODE-IP` with an IP address/FQDN of one of your master nodes.
+Replace `CLUSTER-DEFINITION-PATH` with the path to your existing cluster defintion files that were generated during the intial bootstrap/deployment.
+
+----
+skuba cluster init --control-plane MASTER-NODE-IP /tmp/new-cluster-init
+cp /tmp/new-cluster-init/addons CLUSTER-DEFINITION-PATH
```
Comment (mjura) [inline adoc/MAIN.release-notes.adoc]: I would change it to 
`mv CLUSTER-DEFINITION-PATH/addons CLUSTER-DEFINITION-PATH/addons-old`
`cp -r /tmp/mjura-cluster/addons CLUSTER-DEFINITION-PATH/`

**Review by mjura — CHANGES_REQUESTED** [review]
Only small comment

**Review by mjura — APPROVED** [review]
thank you :+1:

**Comment by jordimassaguerpla** [comment]
Merged in https://github.com/SUSE/caasp-release-notes/pull/47

**Pull request closed without merging**